packages/graphql-serve/README.md

```diff
@@ -76,45 +77,24 @@ $ gqlserve --help
 Usage: gqlserve [options]
 
 Options:
-  --version    Show version number                                     [boolean]
-  --model, -m  Specify the models directory                             [string]
-  --port, -p   Specify the port on which to listen on                   [number]
-  -h, --help   Show help                                               [boolean]
+  --version     Show version number                                    [boolean]
+  --model, -m   Specify the models directory                            [string]
+  --port, -p    Specify the port on which to listen on                  [number]
+  --schemaOnly  Only show GraphQL schema generated from data models    [boolean]
```

Comment:
I'm not sure about using camel case in CLI arguments. I think kebab case would be better. @wtrocki what do you think? Is there any compliance issues with camel case?

Reply:
@craicoverflow I have no issues with making it kebab case

Reply:
I will follow the same format like graphback